app/config/dartlang-pub.yaml

```diff
@@ -36,7 +36,7 @@ productionHosts:
 - pub.dartlang.org
 - pub.dev
 - api.pub.dev
-primaryApiUri: https://pub.dartlang.org/
+primaryApiUri: https://pub.dev/
```

Comment:
We should maybe even remove this abstraction completely?...

Reply:
Yeah, I shall do that in a follow-up PR.